compiler/luci/lang/include/luci/IR/CircleNodeMixins.h

```diff
@@ -67,7 +67,7 @@ template <> class CircleNodeMixin<CircleNodeTrait::Bias>
  *
  * TODO Deprecated this class, and use loco::FixedArity instead
  */
-template <unsigned N, typename Base> class FixedArityNode : public Base
+template <uint32_t N, typename Base> class FixedArityNode : public Base
```

Comment:
We have a rule not to mix files in different `compiler/*`
But I don't know splitting to separate PRs will work with compilation.

Reply:
> We have a rule not to mix files in different `compiler/*`
> But I don't know splitting to separate PRs will work with compilation.

It will, ok - I will split this pr